server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/Enqueuer.java

```diff
@@ -122,7 +122,7 @@ private Mono<Void> publishReferenceToRabbit(MailReference mailReference) throws
             .build();
 
         OutboundMessage data = new OutboundMessage(
-            name.toRabbitExchangeName().asString(),
+            name.   toRabbitExchangeName().asString(),
```

Comment:
Mistake?

Reply:
Oops. How did that happen. Fixed in push.